PeerioTechnologies/peerio-mobile
# fix: share file after readyForDownload

#### Relevant info and issue/PR links 
 https://app.clubhouse.io/peerio/story/12526/mobile-unable-to-share-file-uploaded-from-share-extension
#### Testing instructions  
Upload large file. Until it's loaded, sharing should be disabled from the file actions and file will not appear in 'upload from Peerio files' chat action

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
